projects/core/components/alert/alert.component.ts

```diff
@@ -62,11 +56,7 @@ export class TuiAlertComponent<O, I> implements OnInit {
             return;
         }
 
-        timer(
-            tuiIsNumber(this.autoClose)
-                ? this.autoClose
-                : this.options.defaultAutoCloseTime,
-        )
+        timer(tuiIsNumber(this.autoClose) ? this.autoClose : 3000)
```

Comment:
Why not `this.options.autoCloseTime` ?

```suggestion
        timer(tuiIsNumber(this.autoClose) ? this.autoClose : this.options.autoCloseTime)
```

Reply:
There is already a prop to set autoClose duration. It is basically a second layer level of abstracting this away. It is unnecessary and it is defined in notification options, not in alert options which is wrong.